xptifw/src/xpti_trace_framework.cpp

```diff
@@ -1014,7 +1041,9 @@ XPTI_EXPORT_API void xptiFrameworkFinalize() {
 
   xpti::GFrameworkReferenceCounter--;
   if (xpti::GFrameworkReferenceCounter == 0) {
-    delete &xpti::Framework::instance();
+    xpti::Framework *FW = xpti::Framework::release();
+    if (FW)
+      delete FW;
```

Comment:
Looks like "release" must always be followed by the "delete", so they better be in single function.

Reply:
Done